network_wrangler/transitnetwork.py

```diff
@@ -1030,30 +1033,61 @@ def _apply_transit_feature_change_routing(
             nodes_df = self.road_net.nodes_df.loc[
                 :, [TransitNetwork.STOPS_FOREIGN_KEY, "X", "Y"]
             ]
-            for fk_i in properties["set_stops"]:
-                if fk_i not in existing_fk_ids:
-                    WranglerLogger.info(
-                        "Creating a new stop in stops.txt for node ID: {}".format(fk_i)
-                    )
-                    # Add new row to stops
-                    new_stop_id = str(int(fk_i) + TransitNetwork.ID_SCALAR)
-                    if new_stop_id in stops["stop_id"].tolist():
-                        WranglerLogger.error("Cannot create a unique new stop_id.")
-
-                    stops.loc[
-                        len(stops.index) + 1,
-                        [
-                            "stop_id",
-                            "stop_lat",
-                            "stop_lon",
-                            TransitNetwork.STOPS_FOREIGN_KEY,
-                        ],
-                    ] = [
-                        new_stop_id,
-                        nodes_df.loc[nodes_df[TransitNetwork.STOPS_FOREIGN_KEY] == int(fk_i), "Y"].values[0],
-                        nodes_df.loc[nodes_df[TransitNetwork.STOPS_FOREIGN_KEY] == int(fk_i), "X"].values[0],
-                        fk_i,
-                    ]
+            for trip_id in trip_ids:
+                for fk_i in properties["set_stops"]:
+                    if fk_i in existing_fk_ids:
+                       existing_agency_raw_name = stops[stops[TransitNetwork.STOPS_FOREIGN_KEY]==fk_i]['agency_raw_name'].iloc[0]
+                       existing_trip_id = stops[stops[TransitNetwork.STOPS_FOREIGN_KEY]==fk_i]['trip_id'].iloc[0]
```

Comment:
Shouldn't the trip_id be the trip_id of the changing trip? not just the first trip that uses this stop?